### PR TITLE
Add active styling to dropdown main nav + mobile

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
And refactor a bit.

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
### Desktop

dropdown-active-1

dropdown-active-2

## Mobile

dropdown-active-3

dropdown-active-4

**Comment by RonaldDijkstra** [comment]
Tested and works in IE10+, and the latest greatest of Edge, Chrome, Firefox, Safari, mobile Chrome and mobile Safari.

File: source/stylesheets/components/_dropdown-menu.scss [hunk 1]
```diff
@@ -583,3 +592,5 @@
     }
   }
 }
+
+// Used for mobile nav overlay
```
Comment (sn3p) [inline source/stylesheets/components/_dropdown-menu.scss]: Wut?

File: source/stylesheets/components/_dropdown-menu.scss [hunk 2]
```diff
@@ -246,7 +254,7 @@
       display: block;
       font-size: 1em;
       font-weight: 600;
-      padding: 0.6em 0;
+      padding: 0.6em 0 0.6em 0.6em;
```
Comment (sn3p) [inline source/stylesheets/components/_dropdown-menu.scss]: I prefer (and considered best practice):

```
padding: 0.6em;
padding-right: 0;
```

File: source/stylesheets/components/_dropdown-menu.scss (hunk 1 above)
Comment (RonaldDijkstra) [inline source/stylesheets/components/_dropdown-menu.scss]: Ronnie artifact

**Pull request merged**